src/pages/FrontPage.js

```diff
@@ -43,15 +43,15 @@ export const  FrontPage = React.createClass({
             <ButtonItem color='green' onClick={() => this.displayExercise}>
               Kom i gang!
             </ButtonItem>
-            <ButtonItem color='blue' onClick={() => this.setState({show: !this.state.show})}>
+            <ButtonItem color='blue' onClick={() => this.props.changeMode()}>
```

Comment:
Ja, enig, de endringene skulle kanskje ikke bli tatt med her. Hvordan knappene skal knyttes til teacher/student mode tas i en egen PR, denne er bare for å legge til student mode i redux state three. Slik som det er nå collapser/uncollapser bare Lærer/Veileder knappen infoboksen når du skifter mellom modus. Slik var det ikke tenkt at det skulle være, dette fungerer bare som en placeholder til vil får student mode inn i redux. God idé :+1: Syns å fjerne knappene høres ut som en fin løsning, da får vi fjernet noe visuelt støy som ikke trenger å være der.